improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -790,17 +720,16 @@ def test_weather_data(self):
         cubes[4].data = data_cloud
         cubes[5].data = data_cld_low
         cubes[6].data = data_vis
+        cubes[7].data = data_lightning
```

Comment:
Zeroed the lightning for the alternative data test to keep the expected values unchanged.